.github/workflows/build-and-publish-rule-book.yaml

```
@@ -59,6 +59,11 @@ jobs:
             warning: "Dies ist eine unveröffentlichte Version auf GitHub"
             sections: "sections/translated/de"
             luaengine: true
+          - language: cs
+            version: Verze
+            warning: "Toto je nepublikovaná verze vyrobená z commitu na GitHubu"
+            sections: "sections/translated/cs"
+            luaengine: true
```

```suggestion
            luaengine: false
```
We're using LuaLaTeX only for Cyrillic languages. The German version with it is an error.